Makefile

```diff
@@ -44,6 +42,16 @@ ifndef GCE_PD_CSI_STAGING_VERSION
 endif
 	docker build --build-arg TAG=$(STAGINGVERSION) -t $(STAGINGIMAGE):$(STAGINGVERSION) .
 
+build-push-windows-container:
```

Comment:
Does this invoke gce-pd-driver-windows first?

Reply:
no, it just does the docker build

Reply:
Maybe remove the "build" from the name?

Reply:
here build means docker build.
I see it is used in build the linux container?